cloud/gcp/modules/database/main.tf

```diff
@@ -0,0 +1,47 @@
+resource "random_password" "password" {
+  length           = 40
+  special          = true
+}
+
+
+resource "google_secret_manager_secret" "database_password" {
+  secret_id = "database-password"
+  replication {
+    automatic = true
+  }
+}
+
+resource "google_secret_manager_secret_version" "database_password_version" {
+  secret = google_secret_manager_secret.database_password.id
+  secret_data = random_password.password.result
+}
+
+resource "google_secret_manager_secret_iam_binding" "binding" {
+  secret_id = google_secret_manager_secret.database_password.secret_id
+  role = "roles/secretmanager.secretAccessor"
+  members = [
+    "user:ktoor@google.com",
```

Comment:
can you add a todo here to make this non specific to your user!

Reply:
I have changed this in the next cl

Reply:
your email is hard-coded in https://github.com/seattle-uat/civiform/pull/1881/files as well

Reply:
I have changed it in the next cl. Also, this is just for initial testing. Will product-ionize later